### PR TITLE
hippocrates release notes

### PR DESCRIPTION
**Pull request by bullenc**

### CODE REVIEW

File: docs/API/Release_Notes/API_Release_Notes.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 
 | Version | Date |
 |---|---|
+| [v4.0.0](API_Release_Notes.md#v3280) | July 31, 2023 |
```
Comment (wwysoc2) [inline docs/API/Release_Notes/API_Release_Notes.md:6]: Change anchor to #400

**Review by wwysoc2 — APPROVED** [review]
Approved with one change.

File: docs/API/Release_Notes/API_Release_Notes.md (hunk 1 above)
Comment (wwysoc2) [reply docs/API/Release_Notes/API_Release_Notes.md:6]: * #v400

Comment (bullenc) [reply docs/API/Release_Notes/API_Release_Notes.md]: oops thanks, updated 👍

**Pull request merged**